internal/generator/suite.go

```diff
@@ -40,9 +41,28 @@ func (s *Suite) SetupSuite() {
 	{{ end }}
 	{{ .Cleanup }}
 	{{ .Run }}
+
+	s.RunIncludedSuites()
+}
+
+func (s *Suite) RunIncludedSuites() {
+	{{ .TestIncludedSuites }}
 }
 `
 
+const includedSuiteTemplate = `
+	{{ range .Suites }}
+		s.Run("{{ .Title }}", func() {
+			{{ $suiteName := .Name }}
+			s.{{ $suiteName }}Suite.SetT(s.T())
```

Comment:
Should we call BeforeTest and AfterTest if they are presented?

Reply:
Done